_data-prepper/pipelines/configuration/processors/user-agent.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: default
+title: user_agent
+parent: Processors
+grand_parent: Pipelines
+nav_order: 130
+---
+
+# user_agent
+
+The `user_agent` processor parses any user agent (UA) string in an event then adds the parsing results to the event's write data.
+
+## Usage
+
+In this example, the `user_agent` processor calls the source that contains the UA string, the `ua` field, and indicates the key where the parsed string will write, `user_agent`.
```

Comment:
Is this first example of the `.yml` file illustrating the parsing of the user agent string in an event, as described in the first statement of the "# user_agent" definition above? And then does the example in JSON  illustrate the parsing of the results to the event's write data in the second statement in the "# user_agent" definition? If yes, can you somehow say that this first example performs a first step and the second example performs a second step in the # user_agent" definition? 

Or, is the .yml file example a configuration step while the JSON example fulfills both the parsing and the adding/writing process? 

I guess what I'm asking is better delineation between what the two examples are achieving.

Reply:
This should be resolved now.